caching.md

```diff
@@ -0,0 +1,79 @@
+# Caching in Phoenix applications
+
+## Why?
+
+Despite the
+[speed of Phoenix applications](https://github.com/dwyl/learn-phoenix-framework#why),
+database queries are a bottleneck which can slow down your app. Oftentimes you
+may be loading the same or similar data anytime someone accesses a page. Caching
+previously loaded data can help speed up your app and avoid the query bottleneck!
+
+## What
+
+There are a
+[number of libraries available](https://github.com/h4cc/awesome-elixir#caching)
+for caching data with Phoenix. We are going to be looking at
+[con_cache](https://github.com/sasa1977/con_cache)
+specifically, which is an ETS based key/value storage.
```

Comment:
Only a minor thing, would you be able to add a link like: https://elixirschool.com/lessons/specifics/ets/ ? for ETS, it's just that I hadn't come across it before and it might make people's lives a bit easier who haven't.